styleguide/js/zoolander.js

```diff
@@ -139,6 +141,70 @@ jQuery(document).ready(function($) {
   };
 }(jQuery)); //to protect and scope the JQuery alias = $
 
+//Rackspace Zoolander Tabs jQuery Plugin
+(function($){
+  'use strict';
+  $.fn.rsTabs = function(){
+    var $tabs = $(this);
+    var $zoolanderWindow = $(window);
+
+    $tabs.each(function(){
+      var $thisTabSet = $(this);
+      var $tabLink = $thisTabSet.find('.rsTabs-menu .rsTabs-tabItem');
+
+      //function to reset the tabs on page resizes
+      var resizeTimer;
+      $zoolanderWindow.resize(function(){
+        clearTimeout(resizeTimer);
+        resizeTimer = setTimeout(resizeResetTabs, 250);
```

Comment:
I do like the idea of a timeout here to prevent if from re-firing a bunch.  I'm just wondering about the time frame.  Would it make more sense to set a lower threshhold, like `100` to minimize the delay? More of a question than a suggestion.